scripts/frida/hook_builder.py

```diff
@@ -0,0 +1,100 @@
+import yaml, json
+import sys
+from pathlib import Path
+
+
+def load_template(template_dir, template_name):
+    template_path = template_dir / template_name
+    
+    if not template_path.exists():
+        raise FileNotFoundError(f"Template file not found: {template_path}")
+    
+    with open(template_path, 'r') as f:
+        return f.read()
+
+def generate_java_method_hook(template_dir, package, class_name, method_name, capture_args=True):
+    # TODO: If Jinja is required, I'll identify which templates need merging later
+    # Currently using string replacement
```

Comment:
The templates are so simple at this point that I don't believe we need the overhead of Jinja.